app/scripts/controllers/preferences-controller.test.ts

```diff
@@ -608,6 +608,7 @@ describe('preferences controller', () => {
       expect(controller.state.useAddressBarEnsResolution).toStrictEqual(true);
       expect(controller.state.openSeaEnabled).toStrictEqual(true);
       expect(controller.state.useNftDetection).toStrictEqual(true);
+      expect(controller.state.useSafeChainsListValidation).toStrictEqual(true);
```

Comment:
This was the main fix.

This toggle did not respect basic functionality, and this toggle does not have a UI in the settings pages (so is controlled in the background).